sdk/servicebus/azure-servicebus/tests/test_sb_client.py

```diff
@@ -700,8 +700,9 @@ def test_custom_endpoint_connection_verify_exception(self,
         # invalid cert file to connection_verify should fail
         client = ServiceBusClient(hostname, credential, connection_verify="fakecertfile.pem", uamqp_transport=uamqp_transport)
         with client:
+            sender = client.get_queue_sender(servicebus_queue.name)
             with pytest.raises(ServiceBusError):
-                with client.get_queue_sender(servicebus_queue.name) as sender:
```

Comment:
out of curiosity, why was this change needed?

Reply:
I just wanted to make sure that the error was happening in the same place (when we enter with the context manager), rather than when the object is created. So I separated this into two statements.